requirements-dev.txt

```diff
@@ -5,7 +5,7 @@ click==7.0
 coverage==5.3
 cycler==0.10.0
 decorator==4.4.2
-esse==2021.09.09
+git+ssh://git@github.com/Exabyte-io/esse@43183cad37184896a8ee45e4d85b06863fb21985#egg=esse
 exabyte-json-include==2021.05.06
```

Comment:
works locally.

`git+ssh://git@github.com/Exabyte-io/esse.git@43183cad37184896a8ee45e4d85b06863fb21985#egg=esse`
`esse==git+ssh://git@github.com/Exabyte-io/esse.git@43183cad37184896a8ee45e4d85b06863fb21985#egg=esse`

give the same error
`git+ssh://git@github.com/Exabyte-io/esse.git@43183cad37184896a8ee45e4d85b06863fb21985`
`esse==git+ssh://git@github.com/Exabyte-io/esse.git@43183cad37184896a8ee45e4d85b06863fb21985`
gives a permissions error.